contracts/schemes/Forwarder.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.25;
+
+import "../controller/ControllerInterface.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+/**
+ * @title A scheme to forward a call to a dao.
+ *        The scheme can unregister itself when its expirationTime reached.
+ */
+
+
+contract Forwarder is Ownable {
+
+    Avatar public avatar;
```

Comment:
Why Avatar?
Why not make generic and initialize some address.

Reply:
the avatar is needed to call for unregisterSelf .
we can add another address to make it more generic though I am not sure it is needed.

Reply:
Not needed. This is OK.